composer.json

```diff
@@ -10,8 +10,17 @@
     }
   ],
   "require": {
-    "php": ">=5.6.0"
+    "php": ">=5.6.0",
+    "prestashop/circuit-breaker": "dev-improve-symfony-circuit-breaker",
+    "symfony/cache": "^4.2",
```

Comment:
No problem with the Core `symfony/symfony` package? :thinking: 
Do the Core still using the same Cache class?

Reply:
Good question, I'm gonna check it (I wonder how I can test the version difference, besides comparing new features)
Maybe I need to add an argument in composer so that the module autoload is not prioritary, I don't remember its name

Reply:
https://getcomposer.org/doc/06-config.md#prepend-autoloader :)

Reply:
Thanks! I didn't remember the name of this option ;)

Reply:
Done

Reply:
we have conflicts with guzzle so I can't image we can use Symfony 4 here. More, Symfony 4 require PHP 7.1 so you need to update your minimal PHP version

Reply:
That's weird, I set `"php": ">=5.6.0"` so I guess `symfony/cache: 4.2` is compatible with this version (even though the whole framework isn't) Do you think I should use the same version as in the core?

That aside I was wondering, is it relevant to include dependencies in the module when we know it's already included by the core?